File: CS/Q147080/Tab/Styles/MySkinViewInfoRegistrator.cs
using DevExpress.XtraTab.ViewInfo;
using DevExpress.XtraTab.Registrator;
using DxSample.Tab.ViewInfo;

namespace DxSample.Tab.Styles {
    public class MySkinViewInfoRegistrator : SkinViewInfoRegistrator {
        public MySkinViewInfoRegistrator() : base() { }
        public override BaseTabHeaderViewInfo CreateHeaderViewInfo(BaseTabControlViewInfo viewInfo) {
            return new MySkinTabHeaderViewInfo(viewInfo);
        }
    }
}

File: VB/Q147080/Tab/Styles/MySkinViewInfoRegistrator.vb
Imports DevExpress.XtraTab.ViewInfo
Imports DevExpress.XtraTab.Registrator
Imports DxSample.Tab.ViewInfo

Namespace DxSample.Tab.Styles
    Public Class MySkinViewInfoRegistrator
        Inherits SkinViewInfoRegistrator

        Public Sub New()
            MyBase.New()
        End Sub
        Public Overrides Function CreateHeaderViewInfo(ByVal viewInfo As BaseTabControlViewInfo) As BaseTabHeaderViewInfo
            Return New MySkinTabHeaderViewInfo(viewInfo)
        End Function
    End Class
End Namespace
